### ISSUE TITLE
History issue – back button not working

### ISSUE THREAD

**Issue by yatil** [w3c/wai-wcag-quickref#171]
Steps to reproduce:

* Go to https://www.w3.org/TR/WCAG21/#non-text-content
* Choose the link to "how to meet non-text content"
* Click for example show techniques /failures for 1.1.1
* Then try to click back from browser control

I am not able to go back to WCAG 2.1 spec.

This doesn't happen in WCAG 2.0
https://www.w3.org/TR/WCAG20/#text-equiv

**Comment by yatil** [comment]
Looks like this is a resurfacing of the bugs #134, #132, #128. 

It is interesting that links from WCAG 2.1 are affected, those from WCAG 2.0 are not, so the issue might be with supporting the new permalinks.
